aws/resource_aws_spot_fleet_request.go

```diff
@@ -362,6 +455,28 @@ func buildSpotFleetLaunchSpecification(d map[string]interface{}, meta interface{
 		opts.WeightedCapacity = aws.Float64(wc)
 	}
 
+	if v, ok := d["tag"]; ok && v != "" {
+		tags, _ := spotFleetTagsFromMap(
+			setToMapByKey(v.(*schema.Set), "key"))
+
+		opts.TagSpecifications = []*ec2.SpotFleetTagSpecification{
+			{
+				ResourceType: aws.String(ec2.ResourceTypeInstance),
+				Tags:         tags,
+			},
+		}
+	}
+
+	if v, ok := d["tags"]; ok && v != "" {
+		tags, _ := spotFleetTagsFromList(v.([]interface{}))
+		opts.TagSpecifications = []*ec2.SpotFleetTagSpecification{
+			{
+				ResourceType: aws.String(ec2.ResourceTypeInstance),
+				Tags:         tags,
+			},
+		}
+	}
```

Comment:
It seems by the implementation above that a user cannot specify both `tags` and `tag`. By the looks of it, `tags` would take predecease if both are supplied , is the the intent? If so, they should use the helper/schema `ConflictsWith` attribute. An example of this usage can be found in `aws_instance`:

https://github.com/terraform-providers/terraform-provider-aws/blob/494221fd522c1d41c47790e1a04ce78fcd1c3ab6/aws/resource_aws_instance.go#L106-L110